tfhe/c_api_tests/test_high_level_boolean.c

```diff
@@ -0,0 +1,102 @@
+#include <tfhe.h>
+
+#include <stdio.h>
+#include <inttypes.h>
+#include <assert.h>
+
+#define GOTO_IF_TRUE(label, cond) if ((cond) == 1) {\
+  goto label;\
+  }
```

Comment:
you are more disciplined than me 😛

Reply:
Even tho its probably a good aproach for real code, I replace these with assert to make tracing error easier